monitoring/README.md

```diff
@@ -4,38 +4,67 @@
 
```

Comment:
I would be comforted if the the picture said specifically "MinIO _Server_", because we have used "MinIO" as a shorthand for MinIO _client_ --- and even as a shorthand for the S3 protocol --- in the past.

Reply:
I have seen that when the word "MinIO" is used it implies the server and a clear distinction is used when referring to the MinIO client. For example, see the reference for Minio server and client docker images below:   

- Minio Server:  https://hub.docker.com/r/minio/minio
- Minio Client: 
     i) https://hub.docker.com/r/bitnami/minio-client/
    ii) https://hub.docker.com/r/minio/mc/